css/properties/cursor.json

```diff
@@ -15,7 +15,7 @@
               "version_added": "12"
             },
             "edge_mobile": {
-              "version_added": true
+              "version_added": null
```

Comment:
These should be `false`. `null` means that we don't know and renders as a question mark.

Reply:
I know that, but Microsoft Edge is @erikadoyle’s domain.

Reply:
I'd vote to leave edge mobile as null for now. There are a number of areas that carried over from the desktop codebase to mobile but support details are largely untested since active development was suspended.